.github/dependabot.yml

```diff
@@ -6,3 +6,17 @@ updates:
       interval: "daily"
     allow:
       - dependency-name: "@snapshot-labs/*"
+  - package-ecosystem: "npm"
```

Comment:
The main purpose of removed Dependabot was to reduce noise. Now I found a way to group updates into single PRs and I think we should try it before moving to another solution like Renovate.